docs/09-bootstrapping-kubernetes-workers.md

```diff
@@ -185,6 +185,8 @@ Requires=crio.service
 
 [Service]
 ExecStart=/usr/local/bin/kubelet \\
+  --anonymous-auth=false \\
+  --authorization-mode=Webhook \\
```

Comment:
Ask the API server via the SubjectAccessReview mechanism if the ```subject``` is allowed to access the Kubelet API for a given resource and verb.